client/jetpack-onboarding/connect-success.jsx

```diff
@@ -0,0 +1,52 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React, { Fragment, PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { localize } from 'i18n-calypso';
+import { noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import FormattedHeader from 'components/formatted-header';
+import Tile from 'components/tile-grid/tile';
+import TileGrid from 'components/tile-grid';
+
+class ConnectSuccess extends PureComponent {
+	static propTypes = {
+		href: PropTypes.string,
+		illustration: PropTypes.string,
+		onClick: PropTypes.func,
+		title: PropTypes.string,
+		// from HOCs
+		translate: PropTypes.func,
+	};
+
+	static defaultProps = {
+		onClick: noop,
```

Comment:
I think we don't need the default `onClick` prop. If we remove it, we'll omit the `noop` `onClick` being passed down to the button, and looking at the component tree we should be good to omit it.